src/teststand.cpp

```diff
@@ -62,14 +62,14 @@ void Teststand::initialize()
   can_buses_[0] = std::make_shared<blmc_drivers::CanBus>("can0");
   can_motor_boards_[0] = std::make_shared<blmc_drivers::CanBusMotorBoard>(can_buses_[0]);
 
-  can_buses_[1] = std::make_shared<blmc_drivers::CanBus>("can1");
-  can_motor_boards_[1] = std::make_shared<blmc_drivers::CanBusMotorBoard>(can_buses_[1]);
+  // can_buses_[1] = std::make_shared<blmc_drivers::CanBus>("can1");
```

Comment:
Let us try to find a reasonable way of handling the parametrization of the Testand hardware.